smoker/server/daemon.py

```diff
@@ -164,9 +162,33 @@ def run(self):
             signal.signal(signal.SIGTERM, self._shutdown)
         # API server loop now runs in a separate process and we don't want
         # to terminate to keep an instance of pluginmanager
-        while True:
-            time.sleep(1)
 
+        self._watchdog()
+
+    def _watchdog(self):
+        lg.debug("Starting the smoker watchdog")
+        setproctitle.setproctitle("smokerd watchdog")
```

Comment:
Watchdog is not started in separate process so this will rename original process name, right?
I wouldn't do this and keep it smokerd as the main process should be automatically a watchdog and parent of all plugin processes.